site/_docs/reference.md

```diff
@@ -2689,6 +2689,7 @@ In the following:
 |:- |:-----------------------------------------------|:-----------
 | p | expr :: type                                   | Casts *expr* to *type*
 | m | expr1 <=> expr2                                | Whether two values are equal, treating null values as the same, and it's similar to `IS NOT DISTINCT FROM`
+| p | ACOSD(numeric)                                 | Returns the inverse cosine of *numeric* in degrees
```

Comment:
I personally would like it if the return type was documented as double

Reply:
the behavior for out of range values should also be documented

Reply:
Updated the documentation to say the return type and how out of range values are handled.